provisioner/ansible/test-fixtures/long-debug-message.yml

```diff
@@ -0,0 +1,8 @@
+- name: Stub for Packer testing long Ansible messages
+  hosts: localhost
+  connection: local
+
+  tasks:
+    - name: Very long Ansible output (>65535 chars) (Issue https://github.com/mitchellh/packer/issues/3268)
+      debug:
+        msg: "{{ lipsum(n=300, html=false) }}"
```

Comment:
Apparently, this is built-in function in the jinja templating engine which Ansible uses. I was actually happily surprised to find this function!

I have Ansible installed via pip which installs Jinja as a dep but I don't remember tweaking any python deps so I am pretty sure there is nothing else to be installed